pkg/promtail/targets/journaltarget_test.go

```diff
@@ -110,3 +116,124 @@ func TestJournalTarget(t *testing.T) {
 	assert.Len(t, client.messages, 10)
 	require.NoError(t, jt.Stop())
 }
+
+func TestJournalTarget_Since(t *testing.T) {
+	w := log.NewSyncWriter(os.Stderr)
+	logger := log.NewLogfmtLogger(w)
+
+	initRandom()
+	dirName := "/tmp/" + randName()
+	positionsFileName := dirName + "/positions.yml"
+
+	// Set the sync period to a really long value, to guarantee the sync timer
+	// never runs, this way we know everything saved was done through channel
+	// notifications when target.stop() was called.
+	ps, err := positions.New(logger, positions.Config{
+		SyncPeriod:    10 * time.Second,
```

Comment:
lol, is 10 seconds _really long_ by circle ci standards?

Reply:
hah, I did copy this from another test, but 10 seconds should still be appropriate here I think, unless we've seen individual unit tests taking >10 seconds